homeassistant/components/google_assistant/helpers.py

```diff
@@ -612,6 +612,7 @@ def sync_serialize(self, agent_user_id, instance_uuid):
             "type": get_google_type(
                 state.domain, state.attributes.get(ATTR_DEVICE_CLASS)
             ),
+            "notificationSupportedByAgent": True,
```

Comment:
Need to ve done only if trait exists before we ae done.

Reply:
Can we set this from the trait?

Reply:
Made it conditonal